datacenterlight/views.py

```diff
@@ -49,15 +49,18 @@ def form_invalid(self, form):
 
     def form_valid(self, form):
         form.save()
+        from_emails = {
+            'glasfaser': 'glasfaser@ungleich.ch'
+        }
+        from_page = self.request.POST.get('from_page')
+        print(from_emails.get(from_page, 'info@ungleich.ch'))
         email_data = {
             'subject': "{dcl_text} Message from {sender}".format(
                 dcl_text=settings.DCL_TEXT,
                 sender=form.cleaned_data.get('email')
             ),
             'from_email': settings.DCL_SUPPORT_FROM_ADDRESS,
-            'to': '{}@ungleich.ch'.format(
-                self.request.POST.get('from_page', 'info')
-            ),
+            'to': from_emails.get(from_page, 'info@ungleich.ch'),
```

Comment:
@pcoder using a dict, with key value from the post request. so the email is not directly constructed from the post request value. however I still think it can be done in a better way. any ideas?

Reply:
We can keep this for the moment @tiwariav and perhaps refactor it in a better way later on. But, two points here:
1. The value of `to` key in the `email_data` [needs to be a tuple or a list](https://docs.djangoproject.com/en/1.11/topics/email/#emailmessage-objects)
2. Can we use logger debug instead of print ?